hs_core/models.py

```diff
@@ -736,30 +740,17 @@ def create(cls, **kwargs):
             raise ValidationError("Publisher element can't be created for a resource that "
                                   "is not yet published.")
 
-        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic " \
-                           "Science, Inc. (CUAHSI)"
-
-        if resource.files.all():
```

Comment:
I think this check needs to be kept and an exception be made for the collection resource. Since I implemented this I remember it was a requirement for any resource to be published by Hydroshare it must contain one or more content files. At that time there was no concept of the "Collection" resource. Since Collection resource won't have content files, we need to make that exception only for Collection resource type.